### PR TITLE
Add missing argument which is required

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

File: ambient-light/idlharness.html [hunk 1]
```diff
@@ -98,7 +98,7 @@
   idl_array.add_objects({
     AmbientLightSensor: ['new AmbientLightSensor();'],
     AmbientLightSensorReading: ['new AmbientLightSensorReading({ illuminance: 750 });'],
-    SensorReadingEvent: ['new SensorReadingEvent({ reading: new AmbientLightSensorReading({ illuminance: 750 }) });']
+    SensorReadingEvent: ['new SensorReadingEvent("", { reading: new AmbientLightSensorReading({ illuminance: 750 }) });']
```
Comment (dontcallmedom) [inline ambient-light/idlharness.html]: instead of `""`, I would suggest `"reading"` since that's what would be used in a real event.

Comment (Honry) [inline ambient-light/idlharness.html]: @dontcallmedom, updated, thanks for the suggestion.

**Comment by tobie** [comment]
LGTM

**Comment by zqzhang** [comment]
LGTM, thanks.

**Pull request merged**